atc/api/policychecker/checker_test.go

```diff
@@ -124,7 +121,7 @@ var _ = Describe("PolicyChecker", func() {
 				It("should error", func() {
 					Expect(checkErr).To(HaveOccurred())
 					Expect(checkErr.Error()).To(Equal(`invalid character 'h' looking for beginning of value`))
-					Expect(result).To(BeNil())
+					Expect(result).To(Equal(policy.PolicyCheckResult{}))
```

Comment:
I don't think this is the correct test. In case of error, we should block so this should be:
```suggestion
					Expect(result.Status).To(Equal(policy.Block))
```

Reply:
Actually, this makes me realise that we probably don't need to error out. We should just return the error as the reason for blocking.